src/articles/playbooks/guides/sharing-usage-report.md

```diff
@@ -0,0 +1,108 @@
+---
+uid: playbooks.guides.sharing-usage-report
+---
+
+# Sharing a Customer Usage Report
+
+This is an advanced guide that demonstrates building a scheduled Playbook to share a usage report with your customers every week. This guide leverages Abbot's Skills functionality to pull data from an external source so that it can share information with your customers on a schedule.
+
+Sharing a usage report with customers is a fantastic way to create proactive engagement, drive adoption, and show how your product creates values for your customers.
+
+## Create the Playbook
```

Comment:
Let's make sure all our guides follow this "template". If we want to refine the template, we can do so, but we should update all the guides to be the same.